Cargo.toml

```diff
@@ -30,3 +31,8 @@ winapi = { version = "0.3.9", features = ["processthreadsapi"] }
 
 [target.'cfg(target_os = "redox")'.dependencies]
 redox_syscall = "0.2.1"
+
+[features]
+default = ["locked"]
+tls = []
+locked = ["lazy_static"]
```

Comment:
`locked` is probably not the right name. With this  feature one socket is shared. Rename this feature to `shared`.